### PR TITLE
Remote read api

### PR DESCRIPTION
**Pull request by kushalShukla-web**
Remove the section about remote read JSON responses from Remote_read_api

### CODE REVIEW

**Comment by kushalShukla-web** [comment]
https://github.com/prometheus/prometheus/issues/13975 issue

**Comment by bwplotka** [comment]
Tests are failing, do you mind running `make cli-documentation` and commiting changes?

File: cmd/prometheus/main.go [hunk 1]
```diff
@@ -417,7 +417,7 @@ func main() {
 	serverOnlyFlag(a, "rules.alert.resend-delay", "Minimum amount of time to wait before resending an alert to Alertmanager.").
 		Default("1m").SetValue(&cfg.resendDelay)
 
-	serverOnlyFlag(a, "rules.max-concurrent-evals", "Global concurrency limit for independent rules that can run concurrently.").
+	serverOnlyFlag(a, "rules.max-concurrent-evals", "Global concurrency limit for independent rules that can run concurrently.When changing the value for rules.max-concurrent-evals please adjust the value for query.max-concurrency as the rules manager uses the same query engine than the API:").
```
Comment (bwplotka) [inline cmd/prometheus/main.go:420]: Is this change related to the PR?

File: docs/querying/remote_read_api.md [hunk 2]
```diff
@@ -8,59 +8,6 @@ sort_rank: 7
...
-* `<series_selector>`: Prometheus [time series
-selectors](basics.md#time-series-selectors) like `http_requests_total` or
-`http_requests_total{method=~"(GET|POST)"}` and need to be URL-encoded.
-* `<duration>`: [Prometheus duration strings](basics.md#time_durations).
-For example, `5m` refers to a duration of 5 minutes.
-* `<bool>`: boolean values (strings `true` and `false`).
-
-Note: Names of query parameters that may be repeated end with `[]`.
-
 ## Remote Read API
```
Comment (bwplotka) [inline docs/querying/remote_read_api.md:11]: Perhaps we can remove this line too we have section above.

**Review by bwplotka — COMMENTED** [review]
Thanks! Some comments, almost there.

File: cmd/prometheus/main.go (hunk 1 above)
Comment (kushalShukla-web) [reply cmd/prometheus/main.go:420]: oh sorry i think i have created a new branch from my old existing branch . no these are not related to the issue .

**Comment by kushalShukla-web** [comment]
> Tests are failing, do you mind running `make cli-documentation` and commiting changes?

sure

File: docs/querying/remote_read_api.md (hunk 2 above)
Comment (kushalShukla-web) [reply docs/querying/remote_read_api.md:11]: kk , will do it
